sql/core/src/test/scala/org/apache/spark/sql/test/DataFrameReaderWriterSuite.scala

```diff
@@ -1190,4 +1190,24 @@ class DataFrameReaderWriterSuite extends QueryTest with SharedSparkSession with
     verifyLoadFails(df.write.option("path", path).format("parquet").save(path))
     verifyLoadFails(df.write.option("path", path).format("parquet").save(""))
   }
+
+  test("DataFrameReader.table take the specified options for V1 relation") {
```

Comment:
Shall we add a JIRA prefix in the test title?

Reply:
Sure, done in cc250b9.